src/components/Search.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable max-len */
```

Comment:
- [ ] [OPTIONAL] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: